build/.goreleaser.yml

```diff
@@ -33,6 +27,54 @@ scoop:
     name: Timescale
     email: support@timescale.com
 
+nfpms:
+  - builds: 
+    - promscale
+    formats:
+      - rpm
+      - deb
+
+    file_name_template: "{{ .Binary }}_{{ .RawVersion }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+
+    description: |
+      Promscale is a horizontally scalable and operationally mature analytical
+      platform for Prometheus data that offers the combined power of PromQL
+      and SQL, enabling developers to ask any question, create any dashboard,
+      and achieve greater visibility into their systems.  Promscale is built
+      on top of TimescaleDB, the relational database for time-series built on
+      top of PostgreSQL.
+
+    license: Apache License 2.0
+    maintainer: Timescale <info@timescale.com>
+    contents:
+      - src: build/systemd/promscale.service
+        dst: /etc/systemd/system/promscale.service
+      - src: build/conf/promscale.conf
+        dst: /etc/sysconfig/promscale.conf
+        type: "config|noreplace"
+        packager: rpm
```

Comment:
how does this conf file work on deb then?

Reply:
I somehow missed adding this conf file for deb packaged, will add now.